assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -292,6 +292,11 @@ che.infra.docker.bootstrapper.installer_timeout_sec=180
 # Once servers for one installer available - checks stopped.
 che.infra.docker.bootstrapper.server_check_period_sec=3
 
+# Enable to perform migration of workpace projects at Che startup.
```

Comment:
should it be only for che.infra.docker ? (prefix)

Reply:
Unfortunately, I was not able to protect such a naming when Tyler did choose another naming technique. I don't remember that anyone was supporting me at that point. Does it seem better now?